packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -331,6 +332,29 @@ abstract class _BuildIOSSubCommand extends BuildSubCommand {
       );
     }
 
+    final XCResult xcResult = result.xcResult;
+
+    if (xcResult != null) {
+      if (xcResult.parseSuccess) {
+        if (xcResult.issues.isEmpty) {
+          globals.printStatus('Xcode build has no issues!');
+        }
```

Comment:
Let's remove this trace, we should already be able to see similar output from Xcode in verbose mode.